.gitignore

```
@@ -11,3 +11,6 @@ docs/_build
 __pycache__/
 coverage-html/
 .idea/
+
+# autogenerated by shub deploy
+setup.py
```

This is a very minor thing but I think we should do it more often where it makes sense:

```suggestion
/setup.py
```

To prevent a `setup.py` file anywhere else in the code to be ignored accidentally.